avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -11,7 +11,7 @@ simTypeList = available
 modelType = dfa
 
 #+++++++++++++ Output++++++++++++
-# desired result Parameters (ppr, pfd, pfv, pta, FD, FV, P, Vx, Vy, Vz, TA, particles) - separated by |
+# desired result Parameters (ppr, pfd, pfv, pta, FD, FV, P, FM, Vx, Vy, Vz, TA, particles) - separated by |
```

Comment:
do we need to add this in the docu somewhere?

Reply:
related to #617